packages/genesys-spark-components/src/components/beta/gux-avatar-change-photo/gux-avatar-change-photo.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Component,
+  h,
+  JSX,
+  Element,
+  Event,
+  EventEmitter,
+  Listen
+} from '@stencil/core';
+import { logWarn } from '@utils/error/log-error';
+
+import { buildI18nForComponent, GetI18nValue } from '../../../i18n';
+import defaultResources from './i18n/en.json';
+import { trackComponent } from '@utils/tracking/usage';
+
+/**
+ * @slot avatar - gux-avatar-beta tag
+ */
+
+@Component({
+  styleUrl: 'gux-avatar-change-photo.scss',
+  tag: 'gux-avatar-change-photo-beta',
+  shadow: true
+})
+export class GuxAvatarChangePhoto {
+  private i18n: GetI18nValue;
+
+  @Element() root: HTMLElement;
+
+  @Event()
+  guxchangephoto: EventEmitter<void>;
+
+  @Listen('keydown')
```

Comment:
Since the onclick handler is being used on a `button` the keyboard events are already handled by the native button element. You can remove this `onKeydown` code block